### PR TITLE
feat: optionally exclude descriptions from tag search results

### PR DESCRIPTION
**Pull request by asmfstatoil**

### CODE REVIEW

File: tagreader/odbc_handlers.py [hunk 1]
```diff
@@ -536,7 +540,7 @@ def connect(self):
         self.conn = pyodbc.connect(connection_string, autocommit=True)
         self.cursor = self.conn.cursor()
 
-    def search(self, tag=None, desc=None):
+    def search(self, tag=None, desc=None, return_desc=True):
         query = self.generate_search_query(tag, desc)
```
Comment (asmfstatoil) [inline tagreader/odbc_handlers.py:544]: Not implemented any scrubbing of tag descriptions here

**Comment by einarsi** [comment]
Can you describe the added value in this feature? What is wrong with just doing as described here after getting the results if you are only interested in the tag?
https://github.com/equinor/tagreader-python/issues/87

Also, this PR leaves the search method with two potental output formats: The original `list[tuple[str, str]]` and the new `list[str]`. I am not too keen on that since it can be confusing and can lead to issues. But I am open to being convinced otherwise.

**Comment by asmfstatoil** [comment]
I agree with the proposal in #87 , and actually never want the description when searching for tags. So for me the value is that I do not have to implement the proposed scrubbing in all other applications :) 

For the second point, I thought that the need to specify an optional input value would alert the user that the output format is different.

**Comment by asmfstatoil** [comment]
ODBC is no longer relevant so I think this is good to go

**Comment by asmfstatoil** [comment]
@mortendaehli Can you review?

File: tagreader/odbc_handlers.py (hunk 1 above)
Comment (asmfstatoil) [reply tagreader/odbc_handlers.py]: ODBC is removed so no longer relevant.

**Review by mortendaehli — APPROVED** [review]

**Pull request merged**